tsconfig.build.json

```diff
@@ -10,7 +10,7 @@
     "preserveSymlinks": true,
     "allowJs": true,
     "sourceMap": true,
-    "removeComments": true,
+    "removeComments": false,
```

Comment:
if we remove comments, we don't get useful docstring from the functions when consuming these packages. I think they're a feature?

e.g.

![image](https://user-images.githubusercontent.com/1487463/76692750-ebdfd680-6630-11ea-9b6c-418fa0c37c1b.png)